src/i18n/pt-br/ui.ts

```diff
@@ -103,6 +104,9 @@ export default UIDictionary({
 	'feedback.categoryGroupLabel': 'Escolha a categoria do feedback',
 	'feedback.issue': 'Problema',
 	'feedback.createIssue': 'Criar um problema no GitHub',
+	'feedback.createIssue.description': 'A maneira mais rápida de alertar nosso time sobre um problema.',
+	'feedback.sendFeedback': 'Enviar feedback',
+	'feedback.sendFeedback.description': 'Nos envia um feedback diretamente.',
```

Comment:
Aqui o original é `Send us a message directly`, mudando um pouco fica mais próximo do original.
```suggestion
	'feedback.sendFeedback.description': 'Nos envie uma mensagem diretamente.',
```

Reply:
Going by the translations around this I was thinking that this was for a a description of a button. Now I saw it on the UI, so it makes sense.